core/enclave/src/bridges/ocalls.rs

```diff
@@ -1,38 +1,38 @@
 use sgx_types::*;
-use anonify_types::traits::SliceCPtr;
+use anonify_types::{traits::SliceCPtr, UntrustedStatus};
 use std::vec::Vec;
 use super::auto_ffi::*;
 use crate::error::*;
 
 pub fn get_ias_socket() -> Result<i32> {
-    let mut rt = sgx_status_t::SGX_ERROR_UNEXPECTED;
+    let mut rt = UntrustedStatus::default();
     let mut ias_sock : i32 = 0;
 
     let status = unsafe {
 		ocall_get_ias_socket(
-            &mut rt as *mut sgx_status_t,
+            &mut rt as *mut UntrustedStatus,
             &mut ias_sock as *mut i32
         )
     };
 
     if status != sgx_status_t::SGX_SUCCESS {
 		return Err(EnclaveError::SgxError{ err: status });
 	}
-	if rt != sgx_status_t::SGX_SUCCESS {
-		return Err(EnclaveError::SgxError{ err: rt });
+	if rt != UntrustedStatus::success() {
```

Comment:
I think had better to define `fn is_success(&self) -> bool` to `UntrustedStatus` and then call `rt.is_success()` here.

Reply:
You mean calling `is_err()` ?
I agree with it is more  easy to understand, so I'll fix it (and `EnclaveStatus` in ecall as well).